docs/client-concepts/low-level/getting-started.asciidoc

```diff
@@ -99,10 +99,10 @@ var person = new Person
     LastName = "Laarman"
 };
 
-var indexResponse = lowlevelClient.Index<BytesResponse>("people", "person", "1", PostData.Serializable(person)); <1>
+var indexResponse = lowlevelClient.Index<BytesResponse>("people", "person", "1", PostData.Serializable(person)); <1>

```

Comment:
Would you be able to update the PR to make these changes in the original source file? The source file is:

https://github.com/elastic/elasticsearch-net/tree/6.x/src/Tests/Tests/ClientConcepts/LowLevel/GettingStarted.doc.cs

Reply:
I've made these changes